test/TesterInternal/General/RequestContextTest.cs

```diff
@@ -169,22 +181,33 @@ public void Dispose()
 
             IRequestContextTestGrain grain = this.fixture.GrainFactory.GetGrain<IRequestContextTestGrain>(GetRandomGrainId());
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = activityId;
+#else
             Trace.CorrelationManager.ActivityId = activityId;
+#endif
            Assert.Null(RequestContext.Get(RequestContext.E2_E_TRACING_ACTIVITY_ID_HEADER));
             Guid result = await grain.E2EActivityId();
             Assert.Equal(activityId,  result);  // "E2E ActivityId not propagated correctly"
             RequestContext.Clear();
 
+#if NETSTANDARD
+            RequestContext.ActivityId.Value = nullActivityId;
```

Comment:
the way RequestContext.ActivityId is implemented in net core currently is not supported in full .net. 

I think @dVakulen looked at this problem in #2200 before, maybe he has some thought on this problem.